### ISSUE TITLE
BETTER Analysis units: Control via Thermal Conversion Assumption

### ISSUE THREAD

**Issue by RDmitchell** [SEED-platform/seed#2856]
**Is your feature request related to a problem? Please describe.**
Right now the BETTER analysis is always performed in IP units. It would be nice to allow the user to control whether the analysis was done in IP or SI units

**Describe the solution you'd like**
One possibility would be to use the "Thermal Conversion Assumption" that is in Organizations/Setting/Units. 
![image](https://user-images.githubusercontent.com/6314950/131742660-7d38a96c-46f3-4069-81de-d9efe6122441.png)

**Describe alternatives you've considered**
I don't know of other options, but there might be others. And maybe we need a more universal way to control units.

I am assuming this would control the units of other things such as area (ft2 vs m2), etc, but if it doesn't then this might not be the right solution.

**Comment by macintoshpie** [comment]
I'm wondering if we should have a separate setting for this. Thermal Conversion Assumption affects the way meter data is imported, so maybe we don't want to affect that as well?

**Comment by RDmitchell** [comment]
@macintoshpie -- it does seem like it might be better to have a separate setting, maybe just for BETTER...

**Comment by macintoshpie** [comment]
Yet another option is to add this to the analysis configuration modal. Though this could be frustrating for users if the default value is not what they want (imagine they run an analysis then realize they forgot to switch the units...). This could be solved by another idea of "analysis templates" (rerunnable analysis configurations) but that's probably getting a bit too far ahead of ourselves at the moment.

After thinking about the above it seems most appropriate to have this as a configuration variable for the analysis rather than settings for the organization.

**Comment by RDmitchell** [comment]
Sure

**Comment by RDmitchell** [comment]
@nllong -- what do you think? Seems like it would be nice to be able to switch units, but it's not a high priority. 

P-2 seems good, but seems like we should take the stale label off?

**Comment by nllong** [comment]
Yes, lets keep this around. We should support this unit switching. Also, I think the terms are now USCS (US customary system) and SI. I think this shows the benefit of the stale tool, as it brings these tickets back to our attention!

**Comment by RDmitchell** [comment]
This is still relevant
